presto-parser/src/main/java/com/facebook/presto/sql/SqlFormatter.java

```diff
@@ -831,22 +830,23 @@ protected Void visitCreateTable(CreateTable node, Integer indent)
                 builder.append("\nCOMMENT " + formatStringLiteral(node.getComment().get()));
             }
 
-            appendTableProperties(builder, node.getProperties());
+            process(node.getProperties(), indent);
 
             return null;
         }
 
-        private void appendTableProperties(StringBuilder builder, Map<String, Expression> properties)
+        @Override
+        protected Void visitTableProperties(TableProperties properties, Integer indent)
         {
-            if (!properties.isEmpty()) {
+            if (!properties.getProperties().isEmpty()) {
                 builder.append("\nWITH (\n");
-                // Always output the table properties in sorted order
```

Comment:
Why are you removing the sorting? This was done so that `SHOW CREATE TABLE` would be deterministic. Otherwise, that logic needs to go in every connector. Unit tests rely on this.

We might decide that the formatting should preserve the ordering returned by the connector, but if so, that should be a separate change. (and we'll need to update all the connectors and unit tests which rely on this)